server/apps/microservices/src/processors.ts

```diff
@@ -0,0 +1,45 @@
+import { IAssetUploadedJob, IDeleteFilesJob, IUserDeletionJob, JobName, JobService, QueueName } from '@app/domain';
+import { Process, Processor } from '@nestjs/bull';
+import { Job } from 'bull';
+
+@Processor(QueueName.ASSET_UPLOADED)
+export class AssetUploadedProcessor {
```

Comment:
what is the purpose of breaking down into another layer of abstraction like this?

Reply:
This moved business logic into the domain, which should also make it possible and easier to call the same workflows manually, not to mention test. This is just a binding to auto connect the queues to the public service method.